kolibri/utils/version.py

```diff
@@ -91,7 +100,12 @@ def parse_git_tag_version_string(version_string):
     if len(version_split) >= 2:  # includes a release number (rc, beta, etc.)
         release = version_split[1]
 
-    if len(version_split) >= 3:  # includes a build number and hash
+    if len(version_split) == 3:  # is a full release, includes no release number.
```

Comment:
perhaps this?

```
Running Sphinx v1.4.6
making output directory...

Exception occurred:
  File "/Users/d/Projects/learning_equality/repos/kolibri/kolibri/utils/version.py", line 96, in parse_git_tag_version_string
    build_hash = version_split[3]
IndexError: list index out of range
The full traceback has been saved in /var/folders/n9/k997qvqs3f7_m_310crwsxyc0000gp/T/sphinx-err-jEjQWI.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make[1]: *** [html] Error 1
make: *** [docs] Error 2
```